.github/workflows/pullrequest-verify-chart.yaml

```diff
@@ -0,0 +1,47 @@
+name: Verifies that the chart can be installed, migration runs and the wallet beecomes ready.
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  verify-chart-with-latest-container:
+    name: Verify chart with local built container
+    runs-on: ubuntu-latest
```

Comment:
Should version be pinned? There is a (small) chance that future versions will not support kind, helm and docker...